python/tests/test_network.py

```diff
@@ -3334,7 +3344,7 @@ def test_variable_limits(self):
             self.assertEqual(x.size,net.num_vars)
             for t in range(self.T):
                 for bus in net.buses:
-                    self.assertEqual(x[bus.index_v_mag[t]],bus.v_min)
+                    self.assertEqual(x[bus.index_v_mag[t]],bus.v_min_reg)
```

Comment:
Hmm.  I assume this check is to ensure what these checks are doing. I'll have to look deeper.